aqua/resources-api.aqua

```diff
@@ -88,15 +103,12 @@ func createResource(label: string) -> ?ResourceId, *Error:
             else:
               error <<- res.error
 
-      timeout: ?string
-      success: *bool
-      join successful[0]
-      par timeout <- Peer.timeout(6000, "resource hasn't created: timeout exceeded")
+      success <- wait(successful, 0)
```

Comment:
Waiting for a write to a single peer is worry-some. I suggest to introduce `const REPLICATION_FACTOR=1` and use it.